libs/providers/flagd/package.json

```diff
@@ -7,6 +7,6 @@
   },
   "peerDependencies": {
     "@grpc/grpc-js": "^1.6.0",
-    "@openfeature/js-sdk": "^1.1.0"
+    "@openfeature/js-sdk": ">=1.3.0"
```

Comment:
Require 1.3.0+. @beeme1mr we could actually specify the recent 1.4.0... but technically, 1.3.0 will work.